src/lib/InstantSearch.js

```diff
@@ -82,10 +82,28 @@ Usage: instantsearch({
       this._searchFunction = searchFunction;
     }
 
-    if (urlSync && routing) {
-      throw new Error(
-        'InstantSearch configuration error: it is not possible to use `urlSync` and `routing` at the same time'
```

Comment:
I would still keep the error because the behavior is completely undefined.

Reply:
The error is right here but below (after the warnings)

Reply:
I will put it on top again it's better

Reply:
Sorry, missed it 👍